pkg/inject/oauth_proxy.go

```diff
@@ -18,9 +18,6 @@ import (
 	"github.com/jaegertracing/jaeger-operator/pkg/util"
 )
 
-// #nosec   G101 (CWE-798): Potential hardcoded credentials
-const defaultProxySecret = "ncNDoqLGrayxXzxTn5ANbOXZp3qXd0LA"
```

Comment:
why did it even have a default?

Reply:
the old generation could return an error